docker-compose.yaml

```diff
@@ -9,6 +9,8 @@ services:
       target: base
     ports:
       - '4000:4000'
+    environment:
+      MODE: dev
```

Comment:
I'm guessing that we could use this to enable things like error logs just in dev. Looks useful!